powerauth-restful-model/src/main/java/io/getlime/security/powerauth/rest/api/model/entity/TokenResponsePayload.java

```diff
@@ -19,46 +19,24 @@
  */
 package io.getlime.security.powerauth.rest.api.model.entity;
 
+import lombok.Data;
+
 /**
  * Class representing original payload of the encrypted response for the /pa/token endpoint.
  *
  * @author Petr Dvorak, petr@wultra.com
  */
+@Data
 public class TokenResponsePayload {
 
-    private String tokenId;
-    private String tokenSecret;
-
-    /**
-     * Get token ID.
-     * @return Token ID.
-     */
-    public String getTokenId() {
-        return tokenId;
-    }
-
-    /**
-     * Set token ID.
-     * @param tokenId Token ID.
-     */
-    public void setTokenId(String tokenId) {
-        this.tokenId = tokenId;
-    }
-
     /**
-     * Get token secret.
-     * @return Token secret.
+     * Token ID.
      */
-    public String getTokenSecret() {
-        return tokenSecret;
-    }
+    private String tokenId;
 
     /**
-     * Set token secret.
-     * @param tokenSecret Token secret.
+     * Token Secret.
      */
-    public void setTokenSecret(String tokenSecret) {
-        this.tokenSecret = tokenSecret;
-    }
+    private String tokenSecret;
```

Comment:
I suggest we exclude this field from `toString` to avoid logging the `tokenSecret` by mistake.